lib/registries/addon/components/registries-search-result/template.hbs

```diff
@@ -36,61 +36,68 @@
                     {{if index '| '}}{{source}}
                 {{/each}}
 
-                {{#if result.registrationType}}
-                    | {{result.registrationType}}
+                {{#if @result.registrationType}}
+                    | {{@result.registrationType}}
                 {{/if}}
             </span>
         </div>
 
-        <div local-class="Details {{if expanded 'Expanded' 'Collapsed'}}">
-            <ul local-class="RegistriesSearchResult__HyperLinks" class="m-t-sm">
-                {{#each result.hyperLinks as |hyperlink index|}}
+        <div local-class='Details {{if this.expanded 'Expanded' 'Collapsed'}}'>
+            <ul local-class='RegistriesSearchResult__HyperLinks' class='m-t-sm'>
+                {{#each @result.hyperLinks as |hyperlink index|}}
                     <li>
                         <a
                             href={{hyperlink}}
-                            data-test-result-hyperlink-id="{{result.id}} - {{index}}"
-                            onclick={{action 'click' 'link' (concat 'Discover - Result Hyperlink: ' hyperlink) result.id target=analytics}}
+                            data-test-result-hyperlink-id='{{@result.id}} - {{index}}'
+                            onclick={{action 'click' 'link' (concat 'Discover - Result Hyperlink: ' hyperlink) @result.id target=this.analytics}}
                         >
                             {{hyperlink}}
                         </a>
                     </li>
                 {{/each}}
             </ul>
 
-            <ul local-class="RegistriesSearchResult__InfoLinks">
-                {{#each result.infoLinks as |infolink|}}
+            <ul local-class='RegistriesSearchResult__InfoLinks'>
+                {{#each @result.infoLinks as |infolink|}}
                     <li>
                         <b>{{infolink.type}}: </b>
                         {{infolink.uri}}
                     </li>
                 {{/each}}
             </ul>
 
-            {{#each result.tags as |tag|}}
-                <span class="badge">{{math tag}}</span>
+            {{#each @result.tags as |tag|}}
+                <span class='badge'>{{math tag}}</span>
             {{/each}}
 
         </div>
 
-        <div class="text-center">
-            <button {{action this.toggleExpanded}} data-test-result-toggle-id={{result.id}} local-class="RegistriesSearchResult__Toggle" class="btn btn-link" aria-label={{t 'eosf.components.searchResult.showResult'}}>
-                {{fa-icon footerIcon size=1 ariaHidden=false}}
+        <div class='text-center'>
+            <button
+                data-test-result-toggle-id={{@result.id}}
+                type='button'
+                local-class='RegistriesSearchResult__Toggle'
+                class='btn btn-link'
+                aria-label={{t 'eosf.components.searchResult.showResult'}}
+                {{action this.toggleExpanded}}
+            >
+                {{fa-icon this.footerIcon size=1 ariaHidden=false}}
```

Comment:
Angle bracket invocations here?

Reply:
This feels like scope creep. It's really related to the intent of this PR and without removing all the old invocation style, it does not help much. We could consider enabling [`no-curly-component-invocation`](https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-curly-component-invocation.md) at some point?

Reply:
+1 for `no-curly-component-invocation` in another PR